em-app/src/platform/mod.rs

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) Fortanix, Inc.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#[cfg(feature = "sgx")]
+pub mod sgx;
+
+#[cfg(feature = "sgx")]
+pub use sgx::get_remote_attestation_parameters;
+
+#[cfg(feature = "nitro")]
+pub mod nitro;
+
+#[cfg(feature = "nitro")]
+pub use nitro::get_remote_attestation_parameters;
```

Comment:
They are conflicting features, you can't have both.

Reply:
Features in Rust are meant to be additive. You can easily avoid conflicting features by using a platform specific dependency. Not sure if that works already for Nitro though.

Reply:
@jethrogb I'm proposing dependency injection here to avoid selecting the backend using a feature but we keep arguing about whether that's a good idea here or not. Can you be the tie breaker?

Reply:
Agree with Raoul

Reply:
Removed features, changed to targets as requested.